src/widgets/mfa/MfaCredentialsWidget.jsx

```diff
@@ -7,13 +7,28 @@ import {Info, Intro, Separator} from '../../components/miscComponent';
 import {createForm} from '../../components/form/formComponent';
 import {deepDefaults} from '../../helpers/deepDefaults';
 import phoneNumberField from '../../components/form/fields/phoneNumberField';
+import {withTheme} from '../../components/widget/widgetContext';
+import styled from 'styled-components';
 
 
 const EmailRegisteringCredentialForm = createForm({
     prefix: 'r5-mfa-credentials-email-',
     submitLabel: 'mfa.register.email'
 });
 
+const EmailCredentialRemovalForm = createForm({
+    prefix: 'r5-mfa-credentials-email-removal-',
+    submitLabel: 'mfa.remove.email'
+})
+
+const PhoneNumberCredentialRemovalForm = config => createForm({
+    prefix: 'r5-mfa-credentials-phone-number-removal-',
+    fields: [
+        phoneNumberField({required: true}, config)
+    ],
+    submitLabel: 'mfa.remove.phoneNumber'
+})
+
```

Comment:
Looks like this creates a form to input a phone number? For phone number removal, we already know which phone number is to be removed.